src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.13.0.2104")]
+[assembly: AssemblyVersion("2.13.0.2344")]
```

Comment:
Commenting just to make sure, you revert the changes to this file.

Reply:
Thanks, reverted!